backend/handlers/base_handler.py

```diff
@@ -14,7 +14,7 @@ class BaseHandler(webapp2.RequestHandler):
     def decorateHeaders(self):
         """Decorates headers for the current request."""
         self.response.headers['Access-Control-Allow-Origin'] = '*'
-        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization'
+        self.response.headers['Access-Control-Allow-Headers'] = 'X-API-KEY, Origin, X-Requested-With, Content-Type, Accept, Access-Control-Request-Method, Authorization, Institution-Authorization'
```

Comment:
Nenhum erro  ao enviar convite para novo membro, porem quando este convite é aceito, o adm que enviou o email fica nesse log de erro:
  File "/home/mayza/workspace/web/backend/utils.py", line 70, in _assert
    raise exception(msg)
NotAuthorizedException: Invalid Current Institution! User is not an active member. [só consegui reproduzir uma vez]

Reply:
Ao reenviar um convite:
  File "/home/mayza/workspace/web/backend/handlers/resend_invite_handler.py", line 21, in post
    current_institution = body['currentInstitution']
KeyError: 'currentInstitution'

Reply:
:+1: